tools/hangar/src/generated_test_targets.ts

```diff
@@ -78,7 +78,7 @@ export async function testTest(
 ) {
   const fileMap: Record<string, string> = {};
   const platforms = ["sim"];
-  const args = ["test"];
+  const args = ["test", "--no-stream"];
```

Comment:
Is this arg needed for these tests as well?

Reply:
I was going to say yes since I think these tests include snapshots of stdout/stderr, but since the default logs don't include timestamps anymore I'm down to try snapshotting the new format as long as it's stable in CI. One sec...

Reply:
Mostly I'm curious if we need to handle anything special if users run wing in CI. Even if we have to sanitize the output we should make sure it generally works/looks as expected in CI.